docs/4.0/upgrading.md

```diff
@@ -31,10 +31,12 @@ This will edit (or create) your `composer.json` file.
 
 The following methods were already marked as deprecated is the `3.x` line. They are now removed from the class.
 
-- `Period::next`
-- `Period::previous`
-- `Period::add`
-- `Period::sub`
+| removed methods    | possible replacements                        |
+| ------------------ | -------------------------------------------- |
+| `Period::next`     | `Period::move` with positive duration        |
```

Comment:
@Rarst the more I'm thinking about it the more I think that the solution I provided even if it works for your is not the correct one... 

`Period::next` means instantiating a new `Period` object which abuts the current instance at its ending datepoint with any given duration. Using v4 `Period::move` **will only work if you want the new Period instance to have the same duration as the current one**

The same reasoning can be apply to `Period::previous`